CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Technical - Reorganize test utils/fixtures/expect folders.
 - Technical - Set the latest `forest-express` dependencies version while generating a project.
 - Technical - Reorder in alphabetical order `options` inside `db-analysis-output`.
+- Command Generate - Now generate most of the many to many relationships.
```

Comment:
You should declare precisely the new way many-to-many associations are detected.

Reply:
Better ?

Reply:
Did you change something?